pcap-mode.el

```diff
@@ -112,14 +122,16 @@
     (define-key kmap (kbd "<return>") 'pcap-mode-view-pkt-contents)
     (define-key kmap (kbd "t") 'pcap-mode-toggle-tcp-conversation-view)
     (define-key kmap (kbd "f") 'pcap-mode-set-tshark-filter)
+    (define-key kmap (kbd "F") 'pcap-mode-set-named-filter)
     (define-key kmap (kbd "c") 'pcap-mode-search-frames)
     (define-key kmap (kbd "\C-u f")
       'pcap-mode-set-tshark-single-packet-filter)
     (define-key kmap (kbd "s") 'pcap-mode-set-tshark-single-packet-filter)
     (define-key kmap (kbd "r") 'pcap-mode-reload-file)
+    (define-key kmap (kbd "g") 'pcap-mode-clear-filter)
```

Comment:
I'm happy to see additional functions added to the default keymap.  Please update the README as well.